editors/emacs/Portfile

```diff
@@ -141,9 +112,35 @@ platform darwin {
     }
 }
 
-if {${subport} eq ${name} || ${subport} eq "emacs-devel"} {
+if {$subport eq $name || $subport eq "emacs-devel"} {
```

Comment:
Why remove the brackets?

Reply:
Just to be consistent with how the brackets are used elsewhere and trigger a CI rebuild